net/Socket.hpp

```diff
@@ -207,7 +207,7 @@ class Socket
             static std::once_flag once;
             std::call_once(once,
                            [&]() {
-                               LOG_SYS("Failed setsockopt TCP_NODELAY. Will not report further "
+                               LOG_WRN("Failed setsockopt TCP_NODELAY. Will not report further "
```

Comment:
In this case we will need to put back `strerror(errno)`, since `LOG_WRN` wouldn't do that.
```
LOG_WRN("Failed setsockopt TCP_NODELAY. Will not report further failures to set TCP_NODELAY: " << strerror(errno));
```
(Let clang-format do its thing.)